packages/examples/CipEthernetIP/CipsocketGraphite.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
```

Comment:
package name convention? cip_socket_graphite? I think even directory name needs to be changed. Check that.